mkosi/config.py

```diff
@@ -4165,7 +4165,7 @@ def parse_kernel_module_filter_regexp(p: str) -> str:
         dest="firmware_variables",
         metavar="PATH",
         section="Runtime",
-        parse=config_make_path_parser(constants=("custom", "microsoft", "microsoft-mok")),
+        parse=config_make_path_parser(constants=("custom", "microsoft", "microsoft-mok"), required=False),
```

Comment:
Switching this of generally seems like something that could trip up people, since things that would have been caught at config parsing time, will now be caught only later. Maybe we could warn in the required case only if the path starts with `%`?

Reply:
How are other things that can use specifiers handled?